R/utility_spliceHelper_functions.R

```diff
@@ -20,220 +20,236 @@
 # #alternative exon 3' end,-, ENSG00000165916
 # 'ENST00000591696' # alternative TSS, alternative 3' exon (2), 
 # #alternative 5' exon (1) alternative TES, ,+,ENSG00000141349
-#     )]
-  # subject=rowRanges(seBambu.core)[c('ENST00000378344',
-  #                                   'ENST00000319041',
-  #                                   'ENST00000338530',
-  #                                   'ENST00000338530',
-  #                                   'ENST00000338530',
-  #                                   'ENST00000532718',
-  #                                   'ENST00000263331',
-  #                                   'ENST00000602866',
-  #                                   'ENST00000585361')]
-  # query <- rep(query,2000)
-  # subject <- rep(subject,2000)
-## To be modified after merging Yuk Kei's PR
-annotateSpliceOverlapsByDist <- function(query, subject) {
-  queryStartRng <- ranges(selectStartExonsFromGrangesList(query, exonNumber = 1))
-  subjectStartRng <- ranges(selectStartExonsFromGrangesList(subject, exonNumber = 1))
-  queryEndRng <- ranges(selectEndExonsFromGrangesList(query, exonNumber = 1))
-  subjectEndRng <- ranges(selectEndExonsFromGrangesList(subject, exonNumber = 1))
+############################################################
+# query <- readRDS(system.file("extdata", 
+#     "annotateSpliceOverlapByDist_testQuery.rds",
+#     package = "bambu"))
+# subject <- readRDS(system.file("extdata", 
+#        "annotateSpliceOverlapByDist_testSubject.rds",
+#        package = "bambu"))
+############################################################
+#' annotate splice overlap by distance
+#' @noRd
+compareTranscripts <-function(query, subject) {
```

Comment:
Can you add a complete documentation for the user for this function? This should be accessbile by users, I think it will be very useful!

Reply:
I added a full documentation with arguments, values, and examples.